capabilities.json

```diff
@@ -716,64 +610,55 @@
             }
         },
         "fillPoint": {
-            "displayName": "Fill point",
-            "displayNameKey": "Visual_FillPoint",
             "properties": {
                 "show": {
-                    "displayName": "Fill",
-                    "displayNameKey": "Visual_Fill",
                     "type": {
                         "bool": true
                     }
                 }
             }
         },
         "backdrop": {
-            "displayName": "Backdrop",
-            "displayNameKey": "Visual_Backdrop",
             "properties": {
                 "show": {
-                    "displayName": "Show",
-                    "displayNameKey": "Visual_Show",
                     "type": {
                         "bool": true
                     }
                 },
                 "url": {
-                    "displayName": "Image URL",
-                    "displayNameKey": "Visual_Backdrop_ImageURL",
                     "type": {
                         "text": true
                     }
                 }
             }
         },
         "crosshair": {
-            "displayName": "Crosshair",
-            "displayNameKey": "Visual_Crosshair",
             "properties": {
                 "show": {
-                    "displayName": "Crosshair",
-                    "displayNameKey": "Visual_Crosshair",
                     "type": {
                         "bool": true
                     }
                 }
             }
         },
         "outline": {
-            "displayName": "Outline",
-            "displayNameKey": "Visual_Outline",
             "properties": {
                 "show": {
-                    "displayName": "Outline",
-                    "displayNameKey": "Visual_Outline",
                     "type": {
                         "bool": true
                     }
                 }
             }
         }
     },
-    "supportsMultiVisualSelection": true
+    "supportsMultiVisualSelection": true,
+    "supportsHighlight": true,
+    "supportsKeyboardFocus": true,
+    "privileges": [
+        {
+            "name": "WebAccess",
+            "essential": true,
+            "parameters": ["https://*", "ftp://*"]
```

Comment:
remove ftp

Reply:
resolved in this commit https://github.com/microsoft/powerbi-visuals-enhancedscatter/pull/93/commits/a70c2ed983577cab14d4bb990aad63ea9827608e